llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -456,6 +479,19 @@ void saveModule(std::vector<std::unique_ptr<util::SimpleTable>> &OutTables,
   }
 }
 
+void saveDeviceLibModule(
+    std::vector<std::unique_ptr<util::SimpleTable>> &OutTables, int I,
+    LLVMContext &Context, const std::string &DeviceLibFileName) {
+  SMDiagnostic Err;
+  StringRef DeviceLibLoc = DeviceLibDir;
+  std::string DeviceLibPath = DeviceLibLoc.str() + "/" + DeviceLibFileName;
+  std::unique_ptr<Module> DeviceLibIR =
+      parseIRFile(DeviceLibPath, Err, Context);
```

Comment:
Error handling is missed here.

Reply:
Hi, @maksimsab 
Done.
Thanks very much.